snap/snapcraft.yaml

```diff
@@ -18,6 +18,8 @@ base: core
 apps:
   microk8s:
     command: microk8s.wrapper
+  help:
```

Comment:
This does not look right. If we add the help command, there should be a wrapper for `microk8s help` and the `microk8s.wrapper` should be calling this wrapper.

Reply:
Also if we are to add the help command we should work e bit on the presentation. What do you thing?